### DISCUSSION TITLE
How do I add/append my custom fstab?

### DISCUSSION THREAD

**Discussion by minecraft2048** [rauc/meta-rauc-community#56] [Q&A]
I want to mount my NVMe SSD at boot , so I need to add `/dev/nvme0n1p1 /mnt ext4 defaults 0 0` to the fstab file.

Currently I do that by editing https://github.com/rauc/meta-rauc-community/blob/master/meta-rauc-raspberrypi/recipes-core/base-files/files/fstab , but that modifies the meta-rauc-community layer, so I need to maintain a separate git repo for that.

Is there a way for me to modify the fstab from meta-rauc-community from my own meta layer or local.conf?

**Comment by ejoerns** [comment]
@minecraft2048 You may bbappend the base-files recipe just as `meta-rauc-raspberrypi` does:

https://github.com/rauc/meta-rauc-community/blob/master/meta-rauc-raspberrypi/recipes-core/base-files/base-files_%25.bbappend

Then place your custom fstab file next to your .bbappend. Just make sure your layer has higher prio than `meta-rauc-raspberrypi`.
